flink-connectors/flink-connector-elasticsearch6/pom.xml

```diff
@@ -278,6 +278,9 @@ under the License.
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>2.12.2</version>
 				<configuration>
+					<systemPropertyVariables>
+						<jna.nosys>true</jna.nosys>
```

Comment:
is this only relevant for tests or also in production?

Reply:
I think this can also be a problem in production if you have a wrong JNA library installed.

Reply:
Do we know what exception users may get? Should we document the jna requirements?

Reply:
They get a quite verbose exception which you can see [here](https://issues.apache.org/jira/browse/FLINK-10681). I think it should be clear from this exception what you should do.